### DISCUSSION TITLE
Reading the book / books

### DISCUSSION THREAD

**Discussion by ravichandraveeramachaneni** [rasbt/machine-learning-book#20] [Q&A] [answered]
Hi Sebastian, 

Thank you for a wonderful resource to learn ML. I would like to ask as an absolute beginner to ML, should I start with Python Machine Learning book Edition 3 and then come to the new book "ML with Pytorch & Scikit-learn" or can I directly jump into this book and get everything the old book has ? 

Please let me know. Thanks.

**Comment by rasbt** [comment, marked as answer]
Good question. You can definitely skip "Python Machine Learning book Edition 3" and jump directly into "ML with Pytorch & Scikit-learn". You will get everything that PyML 3 has, and more :). 

(The only reason to consider PyML 3 is if you really must use TensorFlow and have no interest in PyTorch whatsoever).

**Reply by ravichandraveeramachaneni** [reply]
Okay got it.  Thank you so much for the clear answer. I will then start with the new book.
